developer-docs-site/docs/concepts/coin-and-token/aptos-token-overview.md

```diff
@@ -0,0 +1,131 @@
+
+One deficiency of the Ethereum token contract is each token having to deploy individual contract code onto
+a contract account to distinguish it from other tokens even if it simply differs by name.
+Solana's account model enables another pattern where code can be reused so that one generic program operates on
+various data. To create a new token, you could create an account that can mint tokens and more accounts that can
+receive them. The mint account itself uniquely determines the token type instead of contract account, and these are all
+passed as arguments to the one contract deployed to some executable account.
+
+The Aptos token standard shares some similarities with Solana, especially how it covers FT, NFT and SFT in one standard and
+also has a similar generic token contract, which also implicitly defines token standard. Basically, instead of deploying
+a new ERC20 smart contract for each new token, all you need to do is call a function in the contract with necessary
+arguments. Depending on what function you call, the token contract will mint/tranfer/burn/... tokens.
+
+#### Token identification
+Aptos identifies a token by its `TokenId` that includes `TokenDataId` and `property_version`. The
+`property_version` shares the same concept with *Edition Account* in Solana, but there is no explicit
+counterpart in Ethereum as it is not required in any token standard interface.
+
+`TokenDataId` is  a globally unique identifier of token group sharing all the metadata except for
+`property_version`, including token creator address, collection name and token name. In Ethereum,
+the same concept is implemented by deploying a token contract under a unique address so an FT type or
+a collection of NFTs is identified by different contract addresses. In Solana, the similar
+concept for token identifier is implemented as mint account, each of which will represent
+one token type. In Aptos, a creator account can have multiple token types created by giving
+different collections and token names. 
+
+
+#### Token categorization
+it is critical to understand, in Aptos, how to categorize different tokens to expect different sets
+of features:
+- `Fungible Token`: Each FT has one unique `TokenId`, which means tokens sharing the same creator, collection,
+  name and property version are fungible.
+- `Non-Fungible Token`: NFTs always have different `TokenId`s, but it is noted that NFTs belonging to the same
+  collection (by nature also the same creator) will share the same `creator` and `collection`
+  fields in their `TokenDataId`s.
+- `Semi-Fungible Token`: The crypto communities lack a common definition for SFT. The only consensus is
+  SFTs are comparatively new types of tokens that combine the properties of NFTs and FTs.
+  Usually this is realized via the customized logic based on different customized properties.
+
+It's worth noting that Solana has an `Edition` concept that represents an NFT that was copied from a Master Edition NFT.
```

Comment:
Should this go above the *Edition Account* reference above? I think it would make that reference more clear.

Reply:
Yes, and it will be better to put a reference list below the whole doc.
https://docs.metaplex.com/programs/token-metadata/accounts#edition